airbyte-integrations/connectors/source-instagram/source_instagram/client/common.py

```diff
@@ -40,11 +39,7 @@ def log_retry_attempt(details):
         logger.info(f"Caught retryable error after {details['tries']} tries. Waiting {details['wait']} more seconds then retrying...")
 
     def should_retry_api_error(exc):
-        if exc.http_status() == status_codes.TOO_MANY_REQUESTS or (
-            exc.http_status() == status_codes.FORBIDDEN and exc.api_error_message() == "(#4) Application request limit reached"
-        ):
-            return True
-        return False
+        return exc.api_transient_error()
```

Comment:
are you sure that this also applies to rate limiting? I don't see it in docs anywhere. Why not just add this as an OR clause?

Reply:
I found information that this value also applies to Rate Limiting, but now I could not find it. 
To be more safely, I added this check to the general list of errors to be handled using `backoff`.

Reply:
SGTM